database/nostr-lmdb/Cargo.toml

```diff
@@ -3,20 +3,22 @@ name = "nostr-lmdb"
 version = "0.42.0"
 edition = "2021"
 description = "LMDB storage backend for nostr apps"
-authors.workspace = true
-homepage.workspace = true
-repository.workspace = true
-license.workspace = true
+authors = { workspace = true }
```

Comment:
Can you revert these, so leave `<key>.workspace = true`?
The same for the deps in this Cargo.toml